Sekoia.io/main.py

```diff
@@ -79,6 +80,7 @@
     module.register(ReportsGetReport, "get-reports/{uuid}")
     module.register(GetEventFieldCommonValues, "get-event-field-common-values")
     module.register(GetEvents, "get-events")
+    module.register(MergeAssets, "MergeAssets")
```

Comment:
```suggestion
    module.register(MergeAssets, "merge-assets")
```

Reply:
Btw, this names is created automatically when using the sdk